modular_skyrat/modules/colony_fabricator/code/tools/tools.dm

```diff
@@ -0,0 +1,150 @@
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	custom_materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 1.75,
+		/datum/material/silver = HALF_SHEET_MATERIAL_AMOUNT * 1.5,
+		/datum/material/titanium = HALF_SHEET_MATERIAL_AMOUNT,
+	)
+	force = 10
+	throwforce = 8
+	throw_speed = 2
+	throw_range = 3
+	attack_verb_continuous = list("drills", "screws", "jabs", "whacks")
+	attack_verb_simple = list("drill", "screw", "jab", "whack")
+	hitsound = 'sound/items/drill_hit.ogg'
+	usesound = 'sound/items/drill_use.ogg'
+	w_class = WEIGHT_CLASS_SMALL
+	toolspeed = 1
+	random_color = FALSE
+	greyscale_config = null
+	greyscale_config_belt = null
+	greyscale_config_inhand_left = null
+	greyscale_config_inhand_right = null
+
+/obj/item/screwdriver/omni_drill/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/manufacturer_examine, COMPANY_FRONTIER)
+
+/obj/item/screwdriver/omni_drill/get_all_tool_behaviours()
+	return list(TOOL_WIRECUTTER, TOOL_SCREWDRIVER, TOOL_WRENCH)
+
+/obj/item/screwdriver/omni_drill/examine(mob/user)
+	. = ..()
+	. += span_notice("Use <b>in hand</b> to switch configuration.\n")
+	. += span_notice("It functions as a <b>[tool_behaviour]</b> tool.")
+
+/obj/item/screwdriver/omni_drill/update_icon_state()
+	. = ..()
+	switch(tool_behaviour)
+		if(TOOL_SCREWDRIVER)
+			icon_state = initial(icon_state)
+		if(TOOL_WRENCH)
+			icon_state = "[initial(icon_state)]_bolt"
+		if(TOOL_WIRECUTTER)
+			icon_state = "[initial(icon_state)]_cut"
+
+/obj/item/screwdriver/omni_drill/attack_self(mob/user, modifiers)
+	. = ..()
+	if(!user)
+		return
+	var/list/tool_list = list(
+		"Screwdriver" = image(icon = icon, icon_state = "drill"),
+		"Wrench" = image(icon = icon, icon_state = "drill_bolt"),
+		"Wirecutters" = image(icon = icon, icon_state = "drill_cut"),
+	)
+	var/tool_result = show_radial_menu(user, src, tool_list, custom_check = CALLBACK(src, PROC_REF(check_menu), user), require_near = TRUE, tooltips = TRUE)
+	if(!check_menu(user) || !tool_result)
+		return
+	switch(tool_result)
+		if("Wrench")
+			tool_behaviour = TOOL_WRENCH
+			sharpness = NONE
+		if("Wirecutters")
+			tool_behaviour = TOOL_WIRECUTTER
+			sharpness = NONE
+		if("Screwdriver")
+			tool_behaviour = TOOL_SCREWDRIVER
+			sharpness = SHARP_POINTY
+	playsound(src, 'sound/items/change_drill.ogg', 50, vary = TRUE)
+	update_appearance(UPDATE_ICON)
+
+/obj/item/screwdriver/omni_drill/proc/check_menu(mob/user)
+	if(!istype(user))
+		return FALSE
+	if(user.incapacitated() || !user.Adjacent(src))
+		return FALSE
+	return TRUE
+
+// Just a completely normal crowbar except its normal sized and can force doors like jaws of life can
+
+/obj/item/crowbar/large/doorforcer
+	name = "prybar"
+	desc = "A large, sturdy crowbar, painted orange. This one just happens to be tough enough to \
+		survive <b>forcing doors open</b>."
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/tools.dmi'
+	icon_state = "prybar"
+	toolspeed = 1
+	force_opens = TRUE
```

Comment:
I skimmed how the var works, and I am like 50% sure toolspeed affects this, so maybe it should equally be raised up to like 1.3 or so, because it can currently open doors wacky-fast for something that can be printed roundstart, enmasse